recipes/reparation_blast/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 2
+  number: 3
```

Comment:
please reset to 0 if you increase the tool version

Reply:
done, thanks